### DISCUSSION TITLE
How do I customize the dark mode colors?

### DISCUSSION THREAD

**Discussion by pat-mok** [chakra-ui/chakra-ui#9126] [Q&A] [answered]
I have been trying to customise the dark mode colors but with no luck. I understand the new chakra version uses the new codes defineConfig() and createSystem(). Then add to value attribute of Chakra Provider. But it doesn’t seem to work

**Comment by segunadebayo** [comment, marked as answer]
Override the semantic tokens, not the raw palette, and use `_light` / `_dark`.

```ts
createSystem(defaultConfig, {
  theme: {
    semanticTokens: {
      colors: {
        bg: { DEFAULT: { value: { _light: "{colors.white}", _dark: "{colors.gray.900}" } } },
      },
    },
  },
})
```
